### PR TITLE
intro/testing: Prepare for next lesson in Brno

### PR DESCRIPTION
**Pull request by encukou**
- Add to lesson plan.
- Use ANSI escapes for colored terminal output (script included).
- Rewordings & clarifications.
- Add copyright notice and license.
- Remove homework, link to MI-PYT instead.

### CODE REVIEW

**Comment by encukou** [comment]
Merged for today's lesson. Let me know if there's anything to change.

**Comment by hroncok** [comment]
Sorry, i haven't noticed the review request (feel free to ping me for it next time). Will look at this and open a new PR if I want to change something.

File: lessons/intro/testing/index.md [hunk 1]
```diff
@@ -157,23 +172,24 @@ def test_some_holidays(year, month, day):
...
+```ansi
+␛[36m(env)$␛[0m python -m pytest -v
 ...
+tests/test_holidays.py::test_xmas[2015] ␛[32mPASSED␛[0m
+tests/test_holidays.py::test_xmas[2016] ␛[32mPASSED␛[0m
+tests/test_holidays.py::test_xmas[2017] ␛[32mPASSED␛[0m
+tests/test_holidays.py::test_xmas[2033] ␛[31mFAILED␛[0m
+tests/test_holidays.py::test_xmas[2048] ␛[31mFAILED␛[0m
+
+␛[32m␛[1m=========================== 5 passed in 0.25 seconds ===========================␛[0m
```
Comment (hroncok) [inline lessons/intro/testing/index.md]: 3 passed, 2 failed here, will fix.

**Comment by hroncok** [comment]
Other things are OK.

**Comment by encukou** [comment]
Oops! Thanks for the fixup at #85.
I'll ping you next time.

**Pull request merged**